src/components/Books/BookItem/BookItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './BookItem.module.css';
+
+const BookItem = ({ book }) => (
+  <div className={styles.BookItem}>
+    {`Name: ${book.name}, Category: ${book.category}`}
+  </div>
+);
```

Comment:
- [ ] The BookItem component should also have a remove button(no functionality yet) as specified in the requirements.